src/codewhisperer/commands/basicCommands.ts

```diff
@@ -203,3 +192,17 @@ export const notifyNewCustomizationsCmd = Commands.declare(
         notifyNewCustomizations().then()
     }
 )
+
+export const showCodeWhispererQuickPickCommand = 'aws.codewhisperer.quickpick'
```

Comment:
quickpick is an implementation detail, can we name this something more semantic similar to `aws.codecatalyst.openDevEnv` , `aws.codecatalyst.listCommands`, etc.
```suggestion
export const showCodeWhispererActions = 'aws.codewhisperer.listCommands'
```